crates/payload/basic/src/lib.rs

```diff
@@ -673,10 +673,6 @@ impl Drop for Cancelled {
 /// Static config for how to build a payload.
 #[derive(Clone, Debug)]
 pub struct PayloadConfig<Attributes> {
-    /// Pre-configured block environment.
-    pub initialized_block_env: BlockEnv,
-    /// Configuration for the environment.
-    pub initialized_cfg: CfgEnvWithHandlerCfg,
```

Comment:
Same comment here https://github.com/paradigmxyz/reth/pull/10510#issuecomment-2312330192 

@onbjerg @mattsse where would be a good place to move these?

Both the OptimismPayloadBuilder and EthereumPayloadBuilder require these. I assume a good place to move these fields would be to OptimismPayloadBuilderAttributes and EthPayloadBuilderAttributes respectively, unless there is somewhere better/shared?

Reply:
ping on this @mattsse @onbjerg

Reply:
@mattsse @onbjerg PTAL

Reply:
@mattsse @onbjerg PTAL

Reply:
thanks checking asap